src/clojure/clojang/jinterface/otp/messaging.clj

```diff
@@ -153,20 +215,33 @@
   (get-pid [this]
     (.self this))
   (send
-    ([this recip-pid-or-name msg]
-      (.send this recip-pid-or-name msg))
+    ([this ^OtpErlangPid recip-pid ^OtpErlangObject msg]
```

Comment:
See the [comment](https://github.com/oubiwann/clojang/pull/20/files#r51929287) on `exit` for info re: type caveats.